default_metadata/component/checkboxes.json

```
@@ -0,0 +1,9 @@
+{
+  "_id": "definition.checkboxes",
+  "_type": "checkbox",
```

```suggestion
  "_type": "checkboxes",
```

sorry! didn't spot that the last time